Tests/Command/RemoveThumbsCommandTest.php

```diff
@@ -0,0 +1,184 @@
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Filesystem\Tests\FilesystemTestCase;
+
+/**
+ * @author Anton Dyshkant <vyshkant@gmail.com>
+ *
+ * @requires function Symfony\Component\Console\Tester\CommandTester::setInputs
+ */
+final class RemoveThumbsCommandTest extends FilesystemTestCase
+{
+    /**
+     * @var ContainerInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $container;
+
+    /**
+     * @var Application
+     */
+    private $application;
+
+    /**
+     * @var RemoveThumbsCommand
+     */
+    private $command;
+
+    /**
+     * @var CommandTester
+     */
+    private $tester;
+
+    /**
+     * @var Pool|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $pool;
+
+    /**
+     * @var MediaManagerInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $mediaManager;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->container = $this->getMockBuilder(ContainerInterface::class)
+            ->getMock();
+
+        $this->command = new RemoveThumbsCommand();
+        $this->command->setContainer($this->container);
+
+        $this->application = new Application();
+        $this->application->add($this->command);
+
+        $this->tester = new CommandTester($this->application->find('sonata:media:remove-thumbnails'));
+
+        $this->pool = $this->getMockBuilder(Pool::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->mediaManager = $this->getMockBuilder(MediaManagerInterface::class)
+            ->getMock();
+
+        $this->container->expects($this->any())
+            ->method('get')
+            ->will($this->returnCallback(function ($id) {
+                switch ($id) {
+                    case 'sonata.media.pool':
+                        return $this->pool;
+                    case 'sonata.media.manager.media':
+                        return $this->mediaManager;
+                }
+
+                return;
+            }));
+    }
+
+    public function testExecuteWithoutArguments()
+    {
+        $this->filesystem->mkdir($this->workspace.DIRECTORY_SEPARATOR.'foo');
+        $this->filesystem->touch($this->workspace.DIRECTORY_SEPARATOR.'foo'.DIRECTORY_SEPARATOR.'thumb_1_foo.ext');
+        $this->filesystem->touch($this->workspace.DIRECTORY_SEPARATOR.'foo'.DIRECTORY_SEPARATOR.'thumb_2_bar.ext');
+
+        $context = [
+            'providers' => [],
+            'formats' => [],
+            'download' => [],
+        ];
+
+        $formats = [
+            'small' => [],
+        ];
+
+        $filesystem = $this->getMockBuilder(Filesystem::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $fileProvider = $this->getMockBuilder(FileProvider::class)
```

Comment:
please check the other `->getMockBuilder(..foo..)->getMock();`, too and try `createMock()` instead